src/core/qgsapplication.cpp

```diff
@@ -207,32 +207,35 @@ void QgsApplication::init( QString profileFolder )
   }
   else
   {
-    char *prefixPath = getenv( "QGIS_PREFIX_PATH" );
-    if ( !prefixPath )
+    if ( ABISYM( mPrefixPath ).isNull() )
```

Comment:
this is actually the only adding line in this PR.
The following lines are only executed if `mPrefixPath` was not set before by `setPrefixPath` which happens in a 3rd party application

Reply:
I think this should go one level deeper (inside the `if ( !prefixPath )`), so the env variable has the priority

Reply:
@3nids what do you think about this suggestion? would be nice to merge this at some point.

Reply:
@m-kuhn done